demo/features/steps/0454-present-proof.py

```diff
@@ -1,24 +1,13 @@
-from behave import given, when, then
-import json
-from time import sleep
-import time
+from behave import when, then
 
 from bdd_support.agent_backchannel_client import (
     read_proof_req_data,
-    read_presentation_data,
     aries_container_request_proof,
     aries_container_verify_proof,
     agent_container_POST,
     async_sleep,
 )
-from runners.agent_container import AgentContainer
 from runners.support.agent import (
-    CRED_FORMAT_INDY,
-    CRED_FORMAT_JSON_LD,
-    DID_METHOD_SOV,
-    DID_METHOD_KEY,
-    KEY_TYPE_ED255,
-    KEY_TYPE_BLS,
```

Comment:
huh is this just because we auto-cleaned unused imports? not sure why else we are making this change

Reply:
I have a similar reaction to Golda. Why is this stuff no longer needed?